packages/font-icons/src-cs/Telerik.FontIcons/Telerik.FontIcons.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
This is a good target for us (.NET 6.0 is an LTS release) and I suppose that we dismissed the idea of compatibility with earlier versions (e.g. people who have not migrated to this version) or using the package in AJAX in some weird way.

Reply:
I think targeting `netstandard2.1` here will be less problematic. 
Can you share the reasoning behind targeting `net6`?

Reply:
Just a note - this was copied directly from the initial [spike](https://github.com/radkostanev/icons-spike/blob/master/Telerik.FontIcons/Telerik.FontIcons.csproj#L4), and as such, there is not too much thought put into selecting the target here. 
If we see value of targeting `netstandard2.1`, we could update it, given that this is not the final version, rather than a work in progress.

Reply:
AJAX and Kendo.Mvc are the candidates that will require a "lower" target than `net6.0` as .NET 3.x is going out of support in December and we will be releasing this in R1 2023. As .NET 5.0 is out of support, we are left with .NET 6.0 as the lowest supported target for the newer .NET products, but supporting this in AJAX and/or Kendo.MVC will require `netstandard2.1`.

Reply:
е54е437 changed the target to `netstandard2.1` for now, but we could update it again if necessary.

Reply:
hey, is there any update on this?
cc: @pepinho24 @gyankov

Reply:
@pepinho24 @gyankov Any updates?

Reply:
@radkostanev  Sorry for the late response, we render icons on the server in MVC, so this package will be useful. Can we add a `net451` build?

Reply:
> @radkostanev Sorry for the late response, we render icons on the server in MVC, so this package will be useful. Can we add a `net451` build?

Is there a higher .NET version that can be targeted in MVC? Supporting .NET 4.5.1 will require [netstandard1.2](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-1-2).

Reply:
I just tested in several projects and a netstandard2.0 build works for both net framework 4.5 and 4.8 projects, although netstandard2.0 does not officially support anything below 4.6.1. I suppose due to the basic APIs used in the project. So a netstandart2.0 build looks like an option.

Reply:
Let's stick to `netstandard1.1` in this case - we might not use some "higher" APIs now (not very likely to be used in the future, but still), but if we decide on using something, then it will be very easy to break a project with .NET 4.5. So let's be explicit. @gyankov you can open a pull request with the change.